src/OrchardCore.Modules/OrchardCore.Users.TimeZone/ViewModels/UserProfileViewModel.cs

```diff
@@ -0,0 +1,10 @@
+using OrchardCore.Modules;
+
+namespace OrchardCore.Users.TimeZone.ViewModels
+{
+    public class UserProfileViewModel
+    {
+        public string TimeZone { get; set; }
+        public ITimeZone[] TimeZones { get; set; }
```

Comment:
Is there a specific reason you're using `ITimeZone[]` instead of `IList<ITimezone>`? Best practices suggest favoring exposing abstractions rather than concrete types: https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/guidelines-for-collections

Reply:
These timezones are already Ordered by Offset and by TimeZone names from GetTimeZones() I don't see a reason why we should use a IList type for these. An array is more lightweight. Though now thinking about this it should maybe be a ReadOnlyCollection<ITimeZone>.

Reply:
Well, my point wasn't about the actual collection being returned, whether that's an array, a list or a collection. My point was just about the return type siganture. An array implements IList<T>, as does a List<T>. So whether you return an array or a list, the return type could (an in my opinion, should,) be typed as IList<T>.

Reply:
I learned something. I thought the Array was not implementing IList but it is! Then in that context I agree.

Reply:
Changed it since it has no incidence on that ViewModel doing so. I'm going to recheck the ITimeZone[] in the abstraction layer though.